SKPhotoBrowser/SKZoomingScrollView.swift

```diff
@@ -306,14 +306,15 @@ public class SKZoomingScrollView: UIScrollView, UIScrollViewDelegate, SKDetectin
     }
     
     // MARK: - SKDetectingImageViewDelegate
-    func handleImageViewSingleTap(view: UIImageView, touch: UITouch) {
-        if photoBrowser?.areControlsHidden() == false && photoBrowser?.enableSingleTapDismiss == true {
-            photoBrowser?.determineAndClose()
+    func handleImageViewSingleTap(touchPoint: CGPoint) {
+        if photoBrowser!.enableSingleTapDismiss {
+            photoBrowser?.performCloseAnimationWithScrollView(self)
```

Comment:
can you change here to `determineAndClose()`?
method above is correct interface for closing.